### DISCUSSION TITLE
Disable data-rac in all components

### DISCUSSION THREAD

**Discussion by giaokhoa** [adobe/react-spectrum#6343] [General]
I'm using vanilla css and don't want data-rac="" show in all components. I think data-rac="" only need for tailwindcss. Can i disable them?

**Comment by reidbarber** [comment]
Not currently, but I've made a PR for discussion here: https://github.com/adobe/react-spectrum/pull/6349

**Comment by devongovett** [comment]
What's the reason for disabling it? It should not affect anything I don't think...
